src/js/tech/html5.js

```diff
@@ -200,6 +200,40 @@ class Html5 extends Tech {
     });
   }
 
+  /**
+   * Attempt to force override of native video tracks.
+   *
+   * @param {Boolean} override - If set to true native video will be overridden,
+   * otherwise native video will potentially be used.
+   */
+  overrideNativeTracks(override) {
+    const removeTracks = (trackType) => {
+      const props = TRACK_TYPES[trackType];
+      const elTracks = this.el()[props.getterName];
+
+      if (this.trackListeners[props.capitalName]) {
+        this.trackListeners[props.capitalName].forEach(trackListener => {
+          elTracks.removeEventListener(trackListener.eventName, trackListener.listener);
+        });
+      }
+    };
+
+    this.featuresNativeVideoTracks = !override;
+    this.featuresNativeAudioTracks = !override;
+
+    if (!this.trackListeners) {
+      this.trackListeners = [];
+    }
+
+    removeTracks('video');
+    removeTracks('audio');
+
+    this.trackListeners.Video = [];
+    this.trackListeners.Audio = [];
+
+    this.proxyNativeTracks_();
```

Comment:
so basically, everything above will remove listeners and then we call proxyNativeTracks_ so that if we turn native back on it'll run but if we turned it off, proxyNativeTracks_ will exit early and not run?

Reply:
That sounds right to me.